step.yml

```diff
@@ -8,12 +8,9 @@ host_os_tags:
 - osx-10.10
 type_tags:
 - utility
-deps:
-  brew:
-  - name: node
-  apt_get:
-  - name: nodejs
```

Comment:
what if node is not installed? the step manages to install node for osx, but for linux it only installs npm.